pkg/virt-handler/vm_test.go

```diff
@@ -31,6 +31,8 @@ import (
 	"sync"
 	"time"
 
+	"kubevirt.io/kubevirt/pkg/network/driver"
```

Comment:
should go with the kubevirt import group

Reply:
I see we have more than one kubevirt import groups.. o_O

Reply:
I'm not fixing that; moved it to all the other related stuff.

Is it OK now ?